### PR TITLE
Post report to nightingale

### PR DESCRIPTION
**Pull request by gsvr**

### CODE REVIEW

**Comment by gsvr** [comment]
@imsickofmaps Please review.

File: src/utils.js [hunk 1]
```diff
@@ -543,5 +543,83 @@ go.utils = {
...
+        var endpoint = "report/";
+        var payload = {
+            contact_key: contact.key,
+            to_addr: contact.msisdn,
+            categories: go.utils.nightingale_categories(im, contact),
+            location: {
+                point: {
+                    type: "Point",
+                    coordinates: [
+                        contact.extra["location:lon"],
```
Comment (imsickofmaps) [inline src/utils.js]: these should be floats

File: test/fixtures.js [hunk 2]
```diff
@@ -857,5 +857,73 @@ return [
...
+            }
+        },
+
+    // POST REPORT - CORRUPTION, OTHER
+        {
+            'repeatable': true,
+            'request': {
+                'method': 'POST',
+                'headers': {
+                    'Authorization': ['Basic ' + new Buffer('test:test').toString('base64')],
```
Comment (imsickofmaps) [inline test/fixtures.js]: I know this isn't checked by harness at the moment, but for consistency sake, can you make this reflect reality of Token auth?

File: go-app-ussd.js [hunk 3]
```diff
@@ -550,6 +550,127 @@ go.utils = {
...
+                return response;
+            });
+    },
+
+    nightingale_patch: function(im, contact) {
+        var method = "patch";
+        var params = null;
+        var endpoint = "report/" + contact.extra.last_report_id + "/";
+        return go.utils
+            .nightingale_get_report(im, contact)
```
Comment (imsickofmaps) [inline go-app-ussd.js]: You don't need to get the report to patch it, just patch with `{"description": "Blah"}`

File: go-app-ussd.js [hunk 4]
```diff
@@ -550,6 +550,127 @@ go.utils = {
...
+                        parseFloat(contact.extra["location:lat"])
+                    ]
+                }
+            },
+            metadata: {
+                language: contact.extra.lang,
+                status: contact.extra.status || "unregistered",
+                country: contact.extra.country || "unregistered"
+            },
+            description: ""
```
Comment (imsickofmaps) [inline go-app-ussd.js]: Please don't set `description` - then we can use a `None` check up on Nightingale

**Comment by imsickofmaps** [comment]
Two minor things then :+1:

**Pull request closed without merging**